src/Bicep.Core/Semantics/Binder.cs

```diff
@@ -30,12 +30,14 @@ public Binder(INamespaceProvider namespaceProvider, IFeatureProvider features, B
             this.cyclesBySymbol = GetCyclesBySymbol(bicepFile, this.bindings);
 
             this.FileSymbol = new FileSymbol(
+                symbolContext,
                 bicepFile.FileUri.LocalPath,
                 bicepFile.ProgramSyntax,
                 NamespaceResolver,
                 outermostScopes,
                 declarations,
-                bicepFile.FileUri);
+                bicepFile.FileUri,
+                bicepFile.FileKind);
```

Comment:
Is there a reason not to just pass in `bicepFile` at this point, rather than passing in the various components of it?

Reply:
Might as well.